test/WebJobs.Script.Tests.Integration/Host/StandbyManagerTests.cs

```diff
@@ -98,6 +98,8 @@ public async Task StandbyMode_EndToEnd()
             Assert.Equal(2, logLines.Count(p => p.Contains("Executed 'Functions.WarmUp' (Succeeded")));
             Assert.Equal(1, logLines.Count(p => p.Contains("Starting host specialization")));
             Assert.Equal(3, logLines.Count(p => p.Contains($"Starting Host (HostId={_expectedHostId}")));
+            Assert.Equal(3, logLines.Count(p => p.Contains($"Loading functions metadata")));
+            Assert.Equal(1, logLines.Count(p => p.Contains($"Loading proxies metadata")));
```

Comment:
Before the fix, after specialization invalid cached info was returned (I could see we never read proxies again after specialization via a breakpoint in the debugger). I added the logs both because they are useful, and because they facilitate this test :)